cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -163,7 +197,9 @@ def expand_date(df: pyspark.sql.DataFrame) -> pyspark.sql.DataFrame:
         .withColumn("Day", F.dayofmonth(df.Date))
     )
 
-
+# %%
+# We define a function that adds the elapsed times from holidays & promos, the data spanning training & test datasets.
+# This data is returned in a DataFrame.
```

Comment:
Let's not add holidays and promos here, that's known only during the function call. Can you tell what this function does generally, in simple terms?

Reply:
How about 'We define a function that old dates (or elapsed times) to the DataFrame?